webclient/uzivatel/signals.py

```diff
@@ -66,9 +62,6 @@ def create_ident_cely(sender, instance: User, **kwargs):
             instance.old = database_user_query.first()
         else:
             instance.old = None
-    if instance.orcid:
-        match = orcid_pattern.search(instance.orcid)
-        instance.orcid = match.group() if match else None
```

Comment:
@pesikj Validuje se to pak někde jinde? Nebo to není potřeba díky našeptávači?

Reply:
@motyc Autocomplete provádí validaci sám, a to jak při zadávání, tak po odeslání formuláře.